README.md

```diff
@@ -113,21 +146,20 @@ Only one terminal for Limber is needed (unless running the integration suite)
 
 ## Linting and formatting
 
-Linting and formatting are provided by rubocop, prettier and Eslint. I strongly
-recommend checking out editor integrations. Also, using lefthook will help
-ensure that only valid files are committed.
+Linting and formatting are provided by rubocop, prettier, ESlint and erb_lint.
+I strongly recommend checking out editor integrations.
+Also, using lefthook will help ensure that only valid files are committed.
 
-```shell
-# Run rubocop
-bundle exec rubocop
-# Run rubocop with safe autofixes
-bundle exec rubocop -a
-# ESlint
-yarn lint
-# Check prettier formatting
-yarn prettier --check .
-# Fix prettier formatting
-yarn prettier --write .
+There are two built-in rake tasks to run the linters. To just perform a check, run:
+
+```sh
+bundle exec rake lint:check
```

Comment:
Nice 👌 

I think it would still be useful to list the individual commands in case you need to run them independently for whatever reason. I guess people can refer to the rake file but slightly less obvious.